reportsPage.php

```diff
@@ -359,11 +341,143 @@ function drawChart() {
             echo '<div id="piechart" style="width: 1200px; height: 700px; margin: auto;"></div>';
 
         }
-        
-        //<form action="reportsExport.php" method="post" class="export-form">
-        //<input type="hidden" name="action" value="export_donors_over_10000">
-        //<input type="submit" value="Export Donors" class="export-btn">
-        //</form>
+		// Report:Frequncy of Giving Greater than Yearly
```

Comment:
Program Specifications - excellent 
Readability - excellent 
Code Efficiency - excellent 
Documentation - good  *there's a couple places that could be more documented, but for the most part every couple lines of code are described by their functionality in the header comments.
Assigned Tasked - completed *note that for List of Donors whose Frequency of Giving is greater than yearly, no data is being displayed when the report is calculated because the data hardcoded in the database has all contributor's dateOfContribution as some date in 2023.